### ISSUE TITLE
Event parameters

### ISSUE THREAD

**Issue by guyromb** [arnesson/cordova-plugin-firebase#62]
I am trying to log an event to firebase and I am able to receive the event name in firebase console, however, I am unable to see the parameters and filter by them. Any idea what can be the problem?

```
window.FirebasePlugin.logEvent('openScreen', {screenName: 'welcome'});
```

**Comment by robertarnesson** [comment]
see the docs about event params: https://firebase.google.com/docs/analytics/ios/events

only the predefined (by firebase) events show up in the firebase console. to see your custom params you need to enable bigquery and dig through your data there

**Comment by guyromb** [comment]
The events are listed here:
https://support.google.com/firebase/topic/6317484?hl=en&ref_topic=6386699

Will test it now.

**Comment by prantikv** [comment]
Same problem with me i tired using the firebase event params but no luck. The event shows up but cannot filter by the prams.
I am using this to set up audience filter https://support.google.com/firebase/answer/6317522?hl=en&ref_topic=6317489#enable

The docs need to be updated to show the steps needed for event params to show up.

**Comment by mikeislearning** [comment]
Is this still being looked into?
I don't see event params showing up in Firebase at all. Just the event type.

**Comment by prantikv** [comment]
@mikeislearning I belive the issue is with the way we can view data in the firebase dashboard. As mentioned in the docs [here ](https://firebase.google.com/docs/analytics/android/events#log_events) 


> Custom parameters: Custom parameters are not represented directly in your Analytics reports, but they can be used as filters in audience definitions that can be applied to every report. Custom parameters are also included in data exported to BigQuery if your app is linked to a BigQuery project.

see this SO question as well http://stackoverflow.com/questions/37507941/firebase-analytics-custom-events-params

I haven't figured out how to make this work. Do post here and let me know if you get it working.

-Thanks

**Comment by chanphillip** [comment]
Hi everyone.
I was implementing logEvent into my app yesterday. After logging some events with parameters, I went to firebase console to see the log today.
I'm confused since I can see the events, but in audience settings after selecting an event, the parameter list did not show any items.
Then I realized I can type in the parameter name there, but as I don't have enough data at the moment so I could see the result.

Would like to know if anyone could confirm this is how it works. Thanks.
